packages/gtk3.rb

```diff
@@ -75,8 +75,8 @@ def self.patch
     # Use locally build subprojects
     @deps = %w[cairo librsvg]
     @deps.each do |dep|
-      FileUtils.rm_rf "subprojects/#{dep}" if Dir.exist?("subprojects/#{dep}")
-      FileUtils.rm_rf "subprojects/#{dep}.wrap" if File.exist?("subprojects/#{dep}.wrap")
+      FileUtils.rm_rf "subprojects/#{dep}"
+      FileUtils.rm_rf "subprojects/#{dep}.wrap"
```

Comment:
Simplify with:
```
FileUtils.rm_rf ["subprojects/#{dep}", "subprojects/#{dep}.wrap"]
```
?

Reply:
It deletes the submodule projects.

(This change is from rubocop.)